src/BuildingRegistry.Consumer.Address/Projections/AddressKafkaProjection.cs

```diff
@@ -138,6 +139,18 @@ await context
                 var address = await context.AddressConsumerItems.FindAsync(message.AddressPersistentLocalId, cancellationToken: ct);
                 address.Status = AddressStatus.Current;
             });
+
+            When<AddressHouseNumberWasReaddressed>(async (context, message, ct) =>
+            {
+                var destinationHouseNumberAddress = await context.AddressConsumerItems.FindAsync(message.ReaddressedHouseNumber.DestinationAddressPersistentLocalId);
```

Comment:
Can we change the syntax away from `destination` and `source`? This now becomes `new` and `previous`.